src/gc/env/gcenv.os.h

```diff
@@ -388,6 +388,11 @@ class GCToOSInterface
     //  The number of processors
     static uint32_t GetCurrentProcessCpuCount();
```

Comment:
It seems like `GetCurrentProcessCpuCount` should be sufficient to choose workstation (single heap) vs server (a heap per core) GC.